ci-operator/config/openshift/hypershift-logging-operator/openshift-hypershift-logging-operator-main.yaml

```diff
@@ -11,6 +11,25 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
+- as: coverage
+  commands: |
+    export CODECOV_TOKEN=$(cat /tmp/secret/CODECOV_TOKEN)
+    make coverage
+  container:
+    from: src
+  secret:
+    mount_path: /tmp/secret
+    name: hypershift-logging-operator-codecov-token
+- as: publish-coverage
+  commands: |
+    export CODECOV_TOKEN=$(cat /tmp/secret/CODECOV_TOKEN)
+    make coverage
+  container:
+    from: src
+  postsubmit: true
+  secret:
+    mount_path: /tmp/secret
+    name: hypershift-logging-operator-codecov-token
```

Comment:
Have you already update the `hypershift-logging-operator-codecov-token` token in a specific path in Vault?

Reply:
Thanks for the heads up @xiaoyu74 , I'll grab someone who has permission to copy the token